php/EE/Completions.php

```diff
@@ -48,21 +52,20 @@ public function __construct( $line ) {
 				return;
 			}
 		}
-
 		if ( $command->can_have_subcommands() ) {
 			// add completion when command is `ee` and alias isn't set.
-			if ( 'ee' === $command->get_name() && false === $is_alias && false == $is_help ) {
+			if ( 'ee' === $command->get_name() && false === $is_alias && false === $is_help ) {
 				$aliases = \EE::get_configurator()->get_aliases();
 				foreach ( $aliases as $name => $_ ) {
 					$this->add( "$name " );
 				}
 			}
-			foreach ( $command->get_subcommands() as $name => $_ ) {
-				$this->add( "$name " );
+			foreach ( $command->get_subcommands() as $name => $subcommand ) {
+				$this->add( $name . ':' . $subcommand->get_shortdesc() );
```

Comment:
@dhsathiya Is bash completion is also working properly after adding this?

Reply:
no bash will not (not tested but will not). We have to add a shell check for that.
Pseudo:
```
   if zsh
       $this->add( $name . ':' . $subcommand->get_shortdesc() );
   else
       $this->add( $name );
```

Reply:
Bash completion script also needs tweeking as we have site name in `arg #3`.
We can continue our deployment putting only Zsh completion for now.